CMakeLists.txt

```diff
@@ -136,8 +136,10 @@ install(DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/
         FILES_MATCHING REGEX "\\.(mxo|mxe|mxe64)")
 
 # install(CODE [[ message(STATUS "${flucoma-core_SOURCE_DIR}") ]])
-install(DIRECTORY "${flucoma-core_SOURCE_DIR}/AudioFiles/" 
+install(DIRECTORY "${flucoma-core_SOURCE_DIR}/Resources/AudioFiles/" 
         DESTINATION "${MAX_PACKAGE_ROOT}/media")
+install(DIRECTORY "${flucoma-core_SOURCE_DIR}/Resources/DataSets/"
```

Comment:
Copy  to the top level `misc` first and then let the install rule for all the top level folders take care of installing in the final destination. Then it will work for both dev and deploy.

Reply:
but we'd then need to .gitignore the json files in misc 🙄

Reply:
AHHH

Reply:
Thanks